org-formation/050-costs/cur.yaml

```diff
@@ -0,0 +1,24 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: 'Monthly Cost and Usage Report'
+Parameters:
+  Bucket:
+    Type: String
+  Frequency:
+    Type: String
+    Default: "monthly"
+    AllowedValues:
+    - "monthly"
+    - "weekly"
+    - "hourly"
+Resources:
+  MonthlyCostReport:
+    Type: "AWS::CUR::ReportDefinition"
+    Properties:
+      ReportName: !Sub "${AWS::StackName}"
+      ReportVersioning: False
+      TimeUnit: !Ref Frequency
+      Compression: "parquet"
+      Format: "parquet"
+      S3Region: !Ref primaryRegion
+      S3Prefix: !Ref Frequency
```

Comment:
your report name is set to `${resourcePrefix}-CUR-Monthly` which evaluates to `sagebase-CUR-Monthly` so with this prefix it will be `Monthly-sagebase-CUR-Monthly`.  seems kinda redundant, no?

Reply:
Yeah, it is very redundant, especially since the prefix is required but AWS will also add its own prefixes based on the date and frequency, giving us a full s3 path that looks like `cost-and-usage-reports.sagebase.org/Monthly/Monthly/year=2022/month=7/Monthly-00001.parquet` where our prefix parameter is the top-level (first) "Monthly". But I suppose I could change the prefix to "CostReport" or something similar.